handlers_test.go

```diff
@@ -922,3 +922,24 @@ func TestServerResourceDeleteHandlerNotFound(t *testing.T) {
 	}
 	assertEqualSCIMErrors(t, expectedError, scimErr)
 }
+
+func TestServerResourcePatchGroupsHandlerNoContent(t *testing.T) {
```

Comment:
Added test.
Please check it. @di-wu

Reply:
We should actually check if no changes are be made in the PATCH handler.

https://github.com/elimity-com/scim/blob/1bc6b4a9715467679091130ef48e1051ea1f9c21/resource_handler_test.go#L136-L165